languages/csharp/exercises/concept/lists/.meta/Example.cs

```diff
@@ -0,0 +1,73 @@
+using System.Collections.Generic;
+
+public static class Languages
```

Comment:
The only thing that this example does not teach that might be useful for the student is the `Contains` method, to check if an element is in the list. That might be worth considering adding.

Reply:
Added